### PR TITLE
[pl] add index page for polish translation

### PR DESCRIPTION
**Pull request by kdebowski**

### CODE REVIEW

File: pl/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+layout: page
+title: Elixir School
+lang: pl
+---
+
+[![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+
+Lekcje programowania w języku Elixir, inspirowane lekcjami Twittera [Scala School](http://twitter.github.io/scala_school/). 
+
+Dostepne w [Việt ngữ][vi], [汉语][cn], [Español][es], [日本語][jp] oraz [Português][pt].
```
Comment (rubytree) [inline pl/index.md]: `Dostępne`

Comment (doomspork) [inline pl/index.md]: We're thinking about removing this line from the indexes with the exception of English, what do you think?  See #270

Comment (kdebowski) [inline pl/index.md]: As for me it languages in left menu section is sufficient. While creating this index.md for polish translation I wanted to keep version consistent with present ones.

Comment (doomspork) [inline pl/index.md]: @rubytree we've decided to remove these lines from all pages other than English.  Could you remove 11-17?

Comment (doomspork) [inline pl/index.md]: @kdebowski we've decided to remove these lines from all pages other than English.  Could you remove 11-17?

Comment (kdebowski) [inline pl/index.md]: @doomspork sure I will delete those lines. Please tell me if I should close this pull request, make changes and than make again pull request, or delete those lines in next commit?

**Comment by kdebowski** [comment]
@doomspork I've updated polish translation and removed list of languages from index page

**Comment by doomspork** [comment]
Perfect @kdebowski!  Thank you so much for your contribution

**Pull request merged**